README.md

```diff
@@ -42,3 +42,7 @@ const OSPlacesClient = require('@hmcts/os-places-client').OSPlacesClient
 
 new OSPlacesClient('<token here>').lookupByPostcode('SN15NB')
 ```
+
+## Version 1.1.0+
```

Comment:
release notes go in https://github.com/hmcts/os-places-client-node/releases not in the readme